components/ai-character.tsx

```diff
@@ -294,6 +295,58 @@ export const AICharacter = ({ form, hasInstanceAccess, save }: AIFormProps) => {
     save();
   };
 
+  useEffect(() => {
+    const getAIsByCategoryType = async(categoryType : CategoryType) => {
```

Comment:
I'm not clear why this function is needed. We are editing a single AI here but this will trigger to load all AIs for a specific category type. I don't see this function being used anywhere on the page though

Reply:
Not on this page but I have created this as a example for @dmitrytorba to understand, we can remove it later, this basically when we select categories and we need ai's to be filtered

Reply:
Removed!